tests/appsec/test_traces.py

```diff
@@ -247,7 +247,7 @@ def validate_appsec_span_tags(span, appsec_data):
 
 
 @rfc("https://datadoghq.atlassian.net/wiki/spaces/APS/pages/2186870984/HTTP+header+collection")
-@released(dotnet="2.5.1", php_appsec="0.2.2", python="?", ruby="1.0.0.beta1")
+@released(dotnet="2.5.1", php_appsec="0.2.2", python="1.2.3", ruby="1.0.0.beta1")
```

Comment:
```suggestion
@released(dotnet="2.5.1", php_appsec="0.2.2", python="1.3.0", ruby="1.0.0.beta1")
```

The release process for `1.3.0` has already been started, so right now we have `1.3.0rc2` released so any scm version up to date with `1.x` should be above `1.3.0rc2`.


This might get tricky since we won't know what version your change will get released in until it is merged into `1.x`